hrabsence/hrabsence.php

```diff
@@ -235,6 +235,8 @@ function hrabsence_civicrm_install() {
   );
   civicrm_api3('message_template', 'create', $msg_params);
 
+  _hrabsence_add_scheduled_jobs();
```

Comment:
Yeah, I totally agree with you here. To be honest I even tried to do that, but I noticed some of the code here is already duplicated on the extension's upgrader methods. So I was afraid to break something in doing this. Since the extension will soon be removed, I think the global function is good enough and it avoids the duplication between the hook_install and the upgrader